tests/fixtures/lcov.info

```diff
@@ -21,4 +20,4 @@ LH:6
 BRF:0
 BRH:0
 BRDA:3,0,0,0
-end_of_record
\ No newline at end of file
+end_of_record
```

Comment:
Have you had a chance to look at the .lcov life and see if there is a trailing space? I think i originally asked for you to add a trailing space because I thought it was a .py file.. now im not so sure..